packages/landing/analytics/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const EVENTS = {
+  CLICK: {
+    LAUNCH_APP: "click/launch-app",
```

Comment:
Since we have this file I think we can have all the different events here instead of defining them on the spot.

`LAUNCH_APP_NAVBAR_DESKTOP`, `LAUNCH_APP_NAVBAR_MOBILE`, `LAUNCH_APP_HERO`, etc

Reply:
Oh maybe, I like that approach.

My train of thought was around "ok the event is `launch app` and we have different places where we trigger it"

Both works for me to be honest

Reply:
Since we have the file I think we can move here.